app/scripts/controllers/ticketFinder.ctrl.js

```diff
@@ -59,7 +57,7 @@ angular.module('yourStlCourts').controller('TicketFinderCtrl', function (TicketF
         params.licenseState = ctrl.citationCriteria.licenseState;
         break;
       case ctrl.TicketFinderToSelect.LOCATION:
-        params.municipalityNames = Municipalities.translateMapNamesToDatabaseNames(ctrl.citationCriteria.municipalityNames);
+        params.municipalityIds = _.map(ctrl.citationCriteria.municipalities, 'id');
```

Comment:
We should bring in Babel and just use es6 arrows and the normal Array.map(...) and Array.filter(...) for these lodash things.

Not now, obviously, but in the future.